app/common/services/preference-service.ts

```diff
@@ -219,6 +223,10 @@ const _defaultPreferences: IPreferenceStore = {
   showGuide: true,
   showWelcome: true,
   fontsize: "normal",
+
+  // Custom CSS related
+  customCss: "",
+  _insertedCustomCssKey: []
```

Comment:
I cannot understand why we need this?

Reply:
This is an internal storage for all CSS currently inserted into web content. These keys are required when cleaning up or updating the custom CSS. The inserted key is returned from [`contents.insertCSS(css[, options])`](https://www.electronjs.org/docs/latest/api/web-contents#contentsremoveinsertedcsskey), and we use these keys to remove the previously inserted CSS when updating the custom CSS with [`contents.removeInsertedCSS(key)`](https://www.electronjs.org/docs/latest/api/web-contents#contentsremoveinsertedcsskey). 

Please refer
 [changes in `app/main/services/window-process-management-service.ts`](https://github.com/Future-Scholars/paperlib/pull/571/files#diff-480388de18bbacaa38daeea99a61479e4dba30204cbc3738df886a0156e307b2) for detail usages of this field.